src/platform/user/profile/vap-svc/components/AddressField/AddressView.jsx

```diff
@@ -5,18 +5,13 @@ import { formatAddress } from 'platform/forms/address/helpers';
 export default function AddressView({ data: address }) {
   const { street, cityStateZip, country } = formatAddress(address);
 
+  // Not hiding the country in Datadog RUM in case we encounter an issue with an
+  // international address
   return (
-    <div>
-      {street}
-      <br />
-      {cityStateZip}
-
-      {country && (
-        <>
-          <br />
-          {country}
-        </>
-      )}
-    </div>
+    <>
+      <div className="dd-privacy-hidden">{street}</div>
+      <div className="dd-privacy-hidden">{cityStateZip}</div>
+      {country && <div>{country}</div>}
```

Comment:
Could you ask @va-albers about this? In case it was an issue, we could mask instead of hide.

Reply:
Will do!

Reply:
From [platform security review](https://dsva.slack.com/archives/C04KS8710UV/p1690827178254219?thread_ts=1690812900.270629&cid=C04KS8710UV)
> Country information alone is not considered PII and thus is safe to use in this way.

Reply:
Nice